File: codeforces/51.yet_another_two_integers_problem/yatip.py
# Author: José Rodolfo (jric2002)
t = int(input())
k = 10 # k = [1;10]
for i in range(0, t):
  a, b = [int(number) for number in input().split(" ")]
  a = abs(a - b)
  min_mov = 0
  while (a != 0):
    if (a <= 10):
      a = 0
      min_mov += 1
    else:
      min_mov = int(a / k)
      a = a % k
  print(min_mov)

File: codeforces/low_difficulty/800_points/51.yet_another_two_integers_problem_[1409A]/yatip.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int t;
  int a, b, min_mov;
  unsigned short int k = 10; // k = [1;10]
  cin >> t;
  for (unsigned short int i = 0; i < t; i++) {
    min_mov = 0;
    cin >> a >> b;
    a = abs(a - b);
    while (a != 0) {
      if (a <= 10) {
        a = 0;
        min_mov++;
      }
      else {
        min_mov += (a / k);
        a = a % k;
      }
    }
    cout << min_mov << endl;
  }
  return 0;
}